common/src/test/java/io/seata/common/util/ReflectionUtilTest.java

```diff
@@ -0,0 +1,75 @@
+package io.seata.common.util;
+
+import java.io.Serializable;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class ReflectionUtilTest {
+
+    @Test
+    public void testGetClassByName() throws Exception {
+        Assertions.assertEquals(String.class,
+                ReflectionUtil.getClassByName("java.lang.String"));
+    }
+
+    @Test
+    public void testGetFieldValue() throws Exception {
+        Assertions.assertEquals("d",
+                ReflectionUtil.getFieldValue(new DurationUtil(), "DAY_UNIT"));
+
+        Assertions.assertThrows(NoSuchFieldException.class,
+                () -> ReflectionUtil.getFieldValue(new Object(), "A1B2C3"));
+    }
+
+    @Test
+    public void testInvokeMethod() throws Exception {
+        Assertions.assertEquals(0,
+                ReflectionUtil.invokeMethod(new String(), "length"));
```

Comment:
Could you create an actual string eg. "foo" so that the length measured is non-zero?

Reply:
Done